styles/native/ts/core/widgets/linechart.ts

```diff
@@ -0,0 +1,92 @@
+import { brand, font } from "../variables";
+import { LineChartType } from "../../types/widgets";
+
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+
+==========================================================================
+    Line Chart
+
+    Default Class For Mendix Line Chart Widget
+========================================================================== */
+// eslint-disable-next-line @typescript-eslint/camelcase
+export const com_mendix_widget_native_linechart_LineChart: LineChartType = {
+    container: {
+        // All ViewStyle properties are allowed
+    },
+    errorMessage: {
+        // All TextStyle properties are allowed
+        fontFamily: font.family,
+        fontSize: font.sizeSmall,
+    },
+    chart: {
+        // All ViewStyle properties are allowed
+    },
+    gridAndLabelsRow: {
+        // All ViewStyle properties are allowed
+    },
+    gridRow: {
+        // All ViewStyle properties are allowed
+    },
+    gridWrapper: {
+        // All ViewStyle properties are allowed
+    },
+    grid: {
+        padding: { top: 8, right: 8, bottom: 32, left: 32 },
```

Comment:
Imo this can be changed to `paddingTop`, `paddingBottom` etc. Because these are real native styling props.

Reply:
yes use the padding: variables declared by Atlas where possible.

Reply:
Yes, will changed that. Using Atlas variables here isn't wise, since this is about the padding inside the svg canvas. We actually don't know how much padding we need to make long tick value labels on the axes fully visible. Jordan and I are going to brainstorm about a solution for this.

Reply:
is it best to add this padding then to core styling of the widget?

Reply:
What is the outcome of the brainstorm? 
Is padding dynamic here? If not, I think using variables is a good idea

Reply:
We shouldn't reuse spacing variables here, because if the values are changed by users, the axis tick value labels could become invisible creating confusion. We will look for padding calculation in the future, so that devs don't need to set this anymore to make tick value labels visible.